### PR TITLE
Thud

### PR DESCRIPTION
**Pull request by Tofee**

### CODE REVIEW

File: meta-android/recipes-core/busybox/files/mdev-partname.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# Script for mdev's uevent, which will create by-partname symlinks (like udev does)
+
+case "$ACTION" in
+	add|"")
+		ACTION="add"
```
Comment (shr-project) [inline meta-android/recipes-core/busybox/files/mdev-partname.sh]: In which case is ACTION empty and is the value set here used somewhere later?

Comment (Tofee) [reply meta-android/recipes-core/busybox/files/mdev-partname.sh]: Ah, yes, that's a line from upstream's mdev-mounts.sh I took as a basis. It's probably completely useless here -- as well as in the upstream's script...

**Comment by shr-project** [comment]
Merged to active branches with small cosmetic cleanup on top:
   a45920de..669e3a5a  master -> master
   19be640d..fc61ad21  thud -> thud
   0fc5816e..85bbe2e2  warrior -> warrior
and cherry-picked to jansa/sumo to test this in sumo as well:
   f94f3841..62dd5410  jansa/sumo -> jansa/sumo

File: meta-android/recipes-core/initrdscripts/initramfs-boot-android_1.0.bb [hunk 2]
```diff
@@ -12,6 +12,8 @@ SRC_URI = " \
 PACKAGES = "${PN}"
 FILES_${PN} = "/init /machine.conf /distro.conf"
 
+RDEPENDS_${PN} = "e2fsprogs-e2fsck e2fsprogs-resize2fs"
```
Comment (Herrie82) [inline meta-android/recipes-core/initrdscripts/initramfs-boot-android_1.0.bb]: @shr-project I mean here, this probably causes that we don't have udevadm anymore

File: meta-hp/conf/machine/tenderloin.conf [hunk 3]
```diff
@@ -52,8 +52,6 @@ KERNEL_IMAGETYPE = "uImage"
 IMAGE_FSTYPES += "tar.gz"
 IMAGE_ROOTFS_EXTRA_SPACE = "200000"
 
-ANDROID_EXTRA_INITRAMFS_IMAGE_INSTALL = "udev e2fsprogs-e2fsck e2fsprogs-resize2fs"
```
Comment (Herrie82) [inline meta-hp/conf/machine/tenderloin.conf]: Here we still have udev

**Pull request closed without merging**